tests/iDibo_tests.py

```diff
@@ -1,11 +1,24 @@
 from nose.tools import *
```

Comment:
this is not used anymore, you should have it greyed out in PyCharm; unused code should always be removed